services/chat/api.go

```diff
@@ -192,32 +208,138 @@ func (api *API) GetChats(ctx context.Context) (map[string]ChannelGroup, error) {
 			BanList:                 community.Description().BanList,
 			Encrypted:               community.Encrypted(),
 			CommunityTokensMetadata: community.Description().CommunityTokensMetadata,
+			UnviewedMessagesCount:   totalUnviewedMessageCount,
+			UnviewedMentionsCount:   totalUnviewedMentionsCount,
 		}
 
 		for t, i := range community.Images() {
 			chGrp.Images[t] = images.IdentityImage{Name: t, Payload: i.Payload}
 		}
 
-		for _, cat := range community.Categories() {
-			chGrp.Categories[cat.CategoryId] = communities.CommunityCategory{
-				ID:       cat.CategoryId,
-				Name:     cat.Name,
-				Position: int(cat.Position),
-			}
+		result[community.IDString()] = chGrp
+	}
+
+	return result, nil
+}
+
+func (api *API) GetChatsByChannelGroupId(ctx context.Context, channelGroupId string) (*ChannelGroup, error) {
+	pubKey := types.EncodeHex(crypto.FromECDSAPub(api.s.messenger.IdentityPublicKey()))
```

Comment:
```suggestion
	pubKey := types.EncodeHex(api.s.messenger.IdentityPublicKeyCompressed()))
```
I believe we can use `IdentityPublicKeyCompressed()` here

Reply:
It didn't work :shrug: